runatlantis.io/docs/repo-level-atlantis-yaml.md

```diff
@@ -157,6 +157,9 @@ Use the `--parallel-pool-size` to configure the max number of plans and applies
 
 Parallel plans and applies work across both multiple directories and multiple workspaces.
 
+:::Note: When `--parallel-plan` or `--parallel-apply` is set at the server config level, it will be enabled for all repos. This behavior is similar to `automerge` and `--automerge`.
```

Comment:
```suggestion
:::Note: When `--parallel-plan` or `--parallel-apply` is set at the server config level, it will be enabled for all repos. This behavior is similar to `--automerge`.
```

Reply:
Note has been removed as behavior has been fixed.